src/Fable.Import.VSCode.LanguageServer.fs

```diff
@@ -256,7 +257,7 @@ module rec LanguageServer =
         abstract detached: bool with get, set
 
     type ServerOptions =
-        U7<Executable, obj, obj, obj, obj, NodeModule, (unit -> Promise<U4<ChildProcess, StreamInfo, Client.MessageTransports, ChildProcessInfo>>)>
+        U5<Executable, {| run: Executable; debug: Executable |}, {| run: NodeModule; debug: NodeModule |}, NodeModule, (unit -> Promise<U4<ChildProcess, StreamInfo, Client.MessageTransports, ChildProcessInfo>>)>
```

Comment:
Not strictly necessary for this but I think this will be useful for other things later.